cpp/include/Ice/SSL/OpenSSL.h

```diff
@@ -2,23 +2,23 @@
 // Copyright (c) ZeroC, Inc. All rights reserved.
 //
 
-#ifndef ICE_OPENSSL_H
-#define ICE_OPENSSL_H
+#ifndef ICE_SSL_OPENSSL_H
+#define ICE_SSL_OPENSSL_H
 
 #include "Certificate.h"
 
 #include <openssl/pem.h>
 #include <openssl/x509v3.h>
 
-namespace IceSSL::OpenSSL
+namespace Ice::SSL::OpenSSL
```

Comment:
If we keep this public API, I would prefer a prefix (OpenSSLCertificate) over a sub-namespace, just like we did for the authentication options.